packages/plugin-rollup/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@metamask/rollup-plugin-snaps",
```

Comment:
This name is not consistent with the other two plugins (`snaps-browserify-plugin`, `snaps-webpack-plugin`), but this is the convention recommended by Rollup [here](https://rollupjs.org/guide/en/#conventions).